codegenerator/cli/templates/static/codegen/src/Time.res

```diff
@@ -1,8 +1,8 @@
 let resolvePromiseAfterDelay = (~delayMilliseconds) => Utils.delay(delayMilliseconds)
 
 let rec retryAsyncWithExponentialBackOff = async (
-  ~backOffMillis=1000,
-  ~multiplicative=2,
+  ~backOffMillis=100,
+  ~multiplicative=4,
```

Comment:
I think this is a better default. It's safe to change, since it's only used for HyperSync retries.

Previously: `1000 - 2000 - 4000 - 8000 - 16000`
Now: `100 - 400 - 1600 - 6400 - 25600`

Overall similar time for retries, start with a much smaller interval and noticeably increase it for the last one in case if the server is down.

Reply:
I'd like to keep 4 for now.